tools/distrib/python/grpcio_tools/grpc_tools/command.py

```diff
@@ -41,56 +41,27 @@ def _get_resource_file_name(
 
 
 def build_package_protos(package_root, strict_mode=False):
+
     proto_files = []
-    inclusion_root = os.path.abspath(package_root)
-    for root, _, files in os.walk(inclusion_root):
+
+    for root, _, files in os.walk(package_root):
         for filename in files:
             if filename.endswith(".proto"):
-                proto_files.append(
-                    os.path.abspath(os.path.join(root, filename))
-                )
+                proto_files.append(os.path.join(root, filename))
 
     well_known_protos_include = _get_resource_file_name("grpc_tools", "_proto")
 
     for proto_file in proto_files:
         command = [
             "grpc_tools.protoc",
-            "--proto_path={}".format(inclusion_root),
+            "--proto_path={}".format(package_root),
             "--proto_path={}".format(well_known_protos_include),
-            "--python_out={}".format(inclusion_root),
-            "--pyi_out={}".format(inclusion_root),
-            "--grpc_python_out={}".format(inclusion_root),
+            "--python_out={}".format(package_root),
+            "--pyi_out={}".format(package_root),
+            "--grpc_python_out={}".format(package_root),
         ] + [proto_file]
         if protoc.main(command) != 0:
             if strict_mode:
                 raise Exception("error: {} failed".format(command))
             else:
                 sys.stderr.write("warning: {} failed".format(command))
-
-
-class BuildPackageProtos(setuptools.Command):
```

Comment:
I see you deleted this - is it expected? Note we also reference it in README:

https://github.com/grpc/grpc/blob/8e67cb088d3709ae74c1ff31d1655bea6c2b86c0/tools/distrib/python/grpcio_tools/README.rst#usage